test/rekt/features/leases.go

```diff
@@ -32,7 +32,7 @@ func KafkaChannelLease() *feature.Feature {
 	f := feature.NewFeatureNamed("KafkaChannel lease")
 
 	f.Assert("verify lease name", verifyLeaseAcquired(
-		"kafkachannel-controller.knative.dev.eventing-kafka.pkg.channel.consolidated.reconciler.controller.reconciler.00-of-01",
+		"kafka-broker-controller.knative.dev.eventing-kafka-broker.control-plane.pkg.reconciler.channel.v2.reconciler.00-of-01",
```

Comment:
This one should not be modified and instead remap the lease name here https://github.com/knative-extensions/eventing-kafka-broker/blob/49ad678b4a629be09ad40f9bee1a7c360837102a/control-plane/config/eventing-kafka-broker/200-controller/100-config-kafka-leader-election.yaml#L49